bgpd/bgp_mac.c

```diff
@@ -311,6 +311,23 @@ void bgp_mac_del_mac_entry(struct interface *ifp)
 	bgp_mac_remove_ifp_internal(bsm, ifp->name);
 }
 
+bool bgp_mac_exist(struct ethaddr *mac)
+{
+	struct bgp_self_mac lookup;
+	struct bgp_self_mac *bsm;
+	static uint8_t tmp [ETHER_ADDR_STRLEN] = {0};
+
+	if (memcmp(mac, &tmp, ETH_ALEN) == 0)
```

Comment:
Is this check necessary ? Even otherwise, I think this can be moved out of this function.

Reply:
Many EVPN update may contain rmac value to be 0. This check can avoid lookup with 0 values. It is a useful check.